meta-lottie-compose/build.gradle

```diff
@@ -0,0 +1,60 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'maven-publish'
+apply plugin: 'kotlin-kapt'
+
+android {
+    defaultConfig {
+        consumerProguardFiles("meta-lottie-proguard-rules.txt")
+    }
+
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
+
+    buildFeatures {
+        compose true
+    }
+
+    composeOptions {
+        kotlinCompilerExtensionVersion = "1.3.2"
+    }
+}
+
+dependencies {
+    implementation Dependencies.kotlinStdlib
+
+    // Compose
+    implementation platform(Dependencies.composeBom)
+    implementation Dependencies.composeUi
+    implementation Dependencies.composeMaterial
+    implementation Dependencies.composePreview
+    implementation Dependencies.lottieCompose
+}
+
+task androidSourcesJar(type: Jar) {
+    from android.sourceSets.main.java.source
+    classifier "sources"
+}
+
+artifacts {
+    archives androidSourcesJar
+}
```

Comment:
You should use the publishing mechanism built into AGP.

```kotlin
    publishing {
        singleVariant("release") {
            withSourcesJar()
            withJavadocJar()
        }
    }
```

https://github.com/StashInvest/android-navigation-flow/blob/7e11fa8ec9e627ff6f18e84486222135ff3442b2/navigation-flow-dagger/build.gradle#L34

Reply:
Thanks, will take a look.